x-pack/plugins/apm/public/application/index.tsx

```diff
@@ -133,7 +124,10 @@ export const renderApp = (
   });
 
   ReactDOM.render(
-    <ApmAppRoot apmPluginContextValue={apmPluginContextValue} />,
+    <ApmAppRoot
+      apmPluginContextValue={apmPluginContextValue}
+      startDeps={startDeps}
+    />,
```

Comment:
Ideally `startDeps` is part of the plugin context value, similar to plugins. We've not named the properties here appropriately but we'll change that separately from this PR.

Reply:
Sounds right for me, I had the similar thoughts, but decided not to modify your plugin context. Are you OK to have it as it is currently implemented.

Reply:
Sure thing.